roles/ztphub/README.md

```diff
@@ -0,0 +1,63 @@
+ZTP GitOps ACM based role
+=========
+
+This Ansible role installs ACM based ZTP GitOps workflow from scratch
+
+Requirements
+------------
+
+* Access to an OpenShift cluster
+* ACM with MCE installed on Openshift cluster
+* Properly configured kubeconfig file for cluster access
+* Existing CatalogSource.
+* Podman and Ansible installed
+
+Role Variables
+--------------
+
+* `ztphub_working_dir`: (string) Local working directory path for files.
+* `ztphub_gitops_operator_install_channel`: (string) GitOps (ArgoCD) operator channel version (default is `latest`)
+* `ztphub_talm_operator_install_channel`: (string) TALM operator channel version (default is `stable`)
+* `ztphub_image`: (string) Image for ZTP generator (default is `quay.io/openshift-kni/ztp-site-generator`)
+* `ztphub_out_dir`: (string) Directory path for extracting files from ZTP image (default is `{{ ztphub_working_dir }}/out`)
+* `ztp_automatic_update`: (bool) Whether to update automatically operators (`installPlanApproval`) (default is `true`)
+* `ztphub_install_ops_mode`: (string) `Manual` or `Automatic` depends on `ztp_automatic_update`.
```

Comment:
I'm not entirley convinced of how the ztphub_automatic_update and ztphub_install_ops_mode pair work together, but regardless of this, note that the name of the ztphub_automatic_update is wrong (it's missing the "hub" part).